src/styles/clipper.less

```diff
@@ -76,14 +76,13 @@
 }
 
 #closeButtonContainer {
-	position: absolute;
+	float: right;
```

Comment:
What does this do?

Reply:
Using absolute positioning makes the CSS a mess for all of its nested elements, so I just floated this to the right to get it in the right position without making the closeButton and closeButtonIcon do crazy thing